io.asgardeo.tomcat.oidc.sample/src/main/webapp/home.jsp

```diff
@@ -59,38 +64,11 @@
             </div>
             <div class="content">
                 <h2>
-                    Hi <%=name%>
+                    If you see this page, it means that your connection works.<br>
+                    This is the user profile the application will receive.
```

Comment:
Better if we can get the doc team's opinion on the wording here

Reply:
The content is modified as per the suggestions by the DOC team.